analyses/cms-open-data-ttbar/ml_helpers.cpp

```diff
@@ -0,0 +1,74 @@
+#include "fastforest.h"
+#include <cmath>
+#include <assert.h>
+#include <map>
+#include <algorithm>
+#include "ROOT/RVec.hxx"
+
+std::map<std::string, std::vector<int>> get_permutations (std::string jet_labels) {
```

Comment:
this function requires many more comments to help readers understand the logic :)

Reply:
goes in the next commit

Reply:
comments has been added